relayer/chains/cosmos/query.go

```diff
@@ -742,8 +743,13 @@ func (cc *CosmosProvider) QuerySendPacket(
 	srcPortID string,
 	sequence uint64,
 ) (provider.PacketInfo, error) {
+	status, err := cc.QueryStatus(ctx)
+	if err != nil {
+		return provider.PacketInfo{}, err
+	}
+
 	q := sendPacketQuery(srcChanID, srcPortID, sequence)
-	ibcMsgs, err := cc.queryIBCMessages(ctx, cc.log, 1, 1000, q)
+	ibcMsgs, err := cc.queryIBCMessages(ctx, cc.log, 1, 1000, q, !cc.bytesEncodedEvents(zap.NewNop(), status.NodeInfo.Version))
```

Comment:
Similar comment on the `!` operator here, it feels like something could be inverted to eliminate the need for this at the call sites.